### PR TITLE
Codap

### PR DESCRIPTION
**Pull request by schanzer**

### CODE REVIEW

File: lessons/Data-Science/CODAP/codap-filtering/langs/en-us/pages/booleans-with-filter-2.adoc [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+|
+
+@n Click Edit. Change `Pounds=` so that it says `Pounds=32`. What happened? @fitb{}{}
+
+@fitb{}{}
+
+@vspace{1ex}
+
+@n What would be a good name for a Transformer with the expression `Pounds<32`?
```
Comment (schanzer) [inline lessons/Data-Science/CODAP/codap-filtering/langs/en-us/pages/booleans-with-filter-2.adoc:30]: Make "name" bold, to highlight it as a key component of a transformer. Ditto for Purpose Statement

File: lessons/Data-Science/CODAP/codap-filtering/langs/en-us/pages/booleans-with-filter-2.adoc [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+@fitb{}{}
+
+@vspace{1ex}
+
+
+@n What would be a good Purpose Statement for a Transformer with the expression `Pounds<32`? @fitb{}{}
+
+|===
+
+@n With your partner, test out each of the Booleans listed on the table below, using `Pounds` @fitb{1em}{} `32` as the expression. Record the effect of each Boolean in the table below.
```
Comment (schanzer) [inline lessons/Data-Science/CODAP/codap-filtering/langs/en-us/pages/booleans-with-filter-2.adoc:41]: The formatting here feels a little cumbersome to me -- I had to read the instructions and look at the table twice to understand what you meant! I bet you can use the same amount of space by just giving each Boolean expression it's own `@n` number, and one long `@fitb` for kids to write the answer

File: lessons/Data-Science/CODAP/codap-filtering/langs/en-us/pages/codap-filter-transformer.adoc
Comment (schanzer) [inline lessons/Data-Science/CODAP/codap-filtering/langs/en-us/pages/codap-filter-transformer.adoc:1]: "The contract's input - or domain - is a Row." --> which is it? The input or the Domain? Those are not the same thing!

Ditto for Output/Range. Input/Output are specific. Domain/Range are general

File: lessons/Data-Science/CODAP/codap-filtering/langs/en-us/pages/codap-filter-transformer.adoc [hunk 3]
```diff
@@ -48,7 +48,7 @@ Make sure you’re logged into the @starter-file{animals} in CODAP. Select the P
 
 @vspace{1ex}
 
-@n A purpose statement explains the purpose of the expression. Purpose Statements in CODAP begin with either _"Checks..."_  or _"Computes..."_. With your class, write a purpose statement for this expression. Type it into Box 6.
+@n Type a purpose statement for this expression into Box 6.
```
Comment (schanzer) [inline lessons/Data-Science/CODAP/codap-filtering/langs/en-us/pages/codap-filter-transformer.adoc:51]: If they're only typing, why is there a `fitb` here?

**Review by schanzer — COMMENTED** [review]
Overall this looks really good. Some small changes to make, but I'd say it's close to done!

File: lessons/Data-Science/CODAP/codap-filtering/langs/en-us/index.adoc [hunk 4]
```diff
@@ -64,57 +64,73 @@ Check in with students. Did this feel efficient or like a good use of our time?
...
+- What do you Wonder?
+** _Possible responses: I wonder if this Transformer will modify the dataset or create a new dataset. I wonder what that Contract is all about. I wonder what a Boolean is._
 
-Invite students to share their observations. Build enthusiasm for the new possibilities that are now available!
+Wonderings will likely emerge about Contracts and Purpose Statements. You may want to devote a few minutes to discussing these important concepts.
 
-=== Investigate
+- A @vocab{contract} is a statement of the @vocab{domain} (input) and @vocab{range} (output) of an expression. Contracts don’t tell us specific inputs. They tell us the data type of input the expression needs. For example, a Contract wouldn’t say that addition requires "3 and 4". Addition works on more than just those two inputs! Instead, it would tells us that addition requires "two Numbers". When we use a Contract, we plug specific numbers or strings into the expression.
```
[the remark this answers is not in the document]
Comment (retabak) [reply lessons/Data-Science/CODAP/codap-filtering/langs/en-us/index.adoc:101]: fyi, this wording was lifted directly from the Contracts lesson: 

https://bootstrapworld.org/materials/fall2023/en-us/lessons/contracts/index.shtml?pathway=algebra-pyret

Perhaps we should reword there? I'm not sure how to improve this.

**Pull request merged**